src/core/server/index.ts

```diff
@@ -230,6 +230,7 @@ export {
   SavedObjectsMigrationLogger,
   SavedObjectsRawDoc,
   SavedObjectSanitizedDoc,
+  SavedObjectUnsanitizedDoc,
```

Comment:
@rudolf I'm going to stop fighting against this export. It's a parameter type of a public `core` API anyway.